.github/workflows/CI_release.yml

```
@@ -0,0 +1,85 @@
+name: CI on release
+
+on:
+  workflow_dispatch:
+  push:    # Initialize release with 'git push origin tag <tag_name>'
+    tags:  # Actual deployment to testpypi and pypi environments requires review
+      - '20[0-9][0-9].0[1-9].[0-9]+'  # Expected limits: 2000.01.0 - 2099.09.123+
+      - '20[0-9][0-9].1[0-2].[0-9]+'  # Expected limits: 2000.10.0 - 2099.12.123+
+
+jobs:
+  build:
+    name: Build package
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: astral-sh/setup-uv@v5
+      - run: uv build .
```

Just to be consistent with the `uv publish` commands below.
```suggestion
      - run: uv build
```